python312Packages.cssutils: 2.9.0 -> 2.10.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.cssutils is: A CSS Cascading Style Sheets library for Python

meta.homepage for python312Packages.cssutils is: https://github.com/jaraco/cssutils

meta.changelog for python312Packages.cssutils is: https://github.com/jaraco/cssutils/blob/v2.10.1/CHANGES.rst


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 2.10.1 with grep in /nix/store/6gqdshfqh6xkvbdxlkilk4jr88y2sqv4-python3.12-cssutils-2.10.1
- found 2.10.1 in filename of file in /nix/store/6gqdshfqh6xkvbdxlkilk4jr88y2sqv4-python3.12-cssutils-2.10.1

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
11 total rebuild path(s)

11 package rebuild(s)

First fifty rebuilds by attrpath
gnomecast
pretalx
python311Packages.cssutils
python311Packages.inlinestyler
python311Packages.premailer
python311Packages.pycaption
python311Packages.tinycss
python312Packages.cssutils
python312Packages.inlinestyler
python312Packages.pycaption
python312Packages.tinycss
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/6gqdshfqh6xkvbdxlkilk4jr88y2sqv4-python3.12-cssutils-2.10.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.cssutils https://github.com/r-ryantm/nixpkgs/archive/869b4f101b8c16e6abf4da6ba6f068339ca55d85.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/869b4f101b8c16e6abf4da6ba6f068339ca55d85#python312Packages.cssutils
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/6gqdshfqh6xkvbdxlkilk4jr88y2sqv4-python3.12-cssutils-2.10.1
ls -la /nix/store/6gqdshfqh6xkvbdxlkilk4jr88y2sqv4-python3.12-cssutils-2.10.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review --extra-nixpkgs-config '{ allowInsecurePredicate = x: true; }'` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages failed to build:</summary>
  <ul>
    <li>pretalx</li>
    <li>pretalx.dist</li>
    <li>pretalx.static</li>
    <li>python311Packages.inlinestyler</li>
    <li>python311Packages.inlinestyler.dist</li>
    <li>python312Packages.inlinestyler</li>
    <li>python312Packages.inlinestyler.dist</li>
  </ul>
</details>
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>gnomecast</li>
    <li>gnomecast.dist</li>
    <li>python311Packages.cssutils</li>
    <li>python311Packages.cssutils.dist</li>
    <li>python311Packages.premailer</li>
    <li>python311Packages.premailer.dist</li>
    <li>python311Packages.pycaption</li>
    <li>python311Packages.pycaption.dist</li>
    <li>python311Packages.tinycss</li>
    <li>python311Packages.tinycss.dist</li>
    <li>python312Packages.cssutils</li>
    <li>python312Packages.cssutils.dist</li>
    <li>python312Packages.pycaption</li>
    <li>python312Packages.pycaption.dist</li>
    <li>python312Packages.tinycss</li>
    <li>python312Packages.tinycss.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dotlambda for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

dotlambda commented on pkgs/development/python-modules/cssutils/default.nix:
```diff
@@ -16,15 +16,15 @@
 
   disabled = pythonOlder "3.8";
 
   format = "pyproject";
```
```suggestion
  pyproject = true;
```

dotlambda committed:
python312Packages.cssutils: 2.9.0 -> 2.10.1

Diff: https://github.com/jaraco/cssutils/compare/refs/tags/v2.9.0...v2.10.1

Changelog: https://github.com/jaraco/cssutils/blob/refs/tags/v2.10.1/NEWS.rst
